WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsJetpackSocialContainer.kt

```diff
@@ -0,0 +1,122 @@
+package org.wordpress.android.ui.posts
+
+import android.content.res.Configuration
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Divider
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import org.wordpress.android.models.PublicizeConnection
+import org.wordpress.android.models.PublicizeConnectionList
+import org.wordpress.android.ui.compose.components.TrainOfIconsModel
+import org.wordpress.android.ui.compose.components.buttons.PrimaryButton
+import org.wordpress.android.ui.compose.theme.AppThemeEditor
+import org.wordpress.android.ui.compose.unit.Margin
+import org.wordpress.android.ui.posts.social.PostSocialConnection
+import org.wordpress.android.ui.posts.social.compose.PostSocialMessageItem
+import org.wordpress.android.ui.posts.social.compose.PostSocialSharesText
+import org.wordpress.android.ui.publicize.PublicizeServiceIcon
+
+@Composable
+fun EditPostSettingsJetpackSocialContainer(
+    modifier: Modifier = Modifier,
```

Comment:
⚠️ The order of the arguments seems swapped, `modifier` should be the first **optional** parameter in the Composable function, as noted in the [documentation](https://developer.android.com/reference/kotlin/androidx/compose/ui/Modifier#:~:text=It%20should%20appear%20as%20the%20first%20optional%20parameter%20in%20the%20parameter%20list%3B%20after%20all%20required%20parameters%20(except%20for%20trailing%20lambda%20parameters)%20but%20before%20any%20other%20parameters%20with%20default%20values.):
> It should appear as the first optional parameter in the parameter list; after all required parameters (except for trailing lambda parameters) but before any other parameters with default values.

So in this case it should be the last parameter in the function since it's the only optional parameter and the lambda parameters are not really meant to be trailing lambda parameters.

Reply:
Good point. Updated 👍